megaqc/cli.py

```diff
@@ -28,10 +28,9 @@ def create_megaqc_app(info):
 @click.group(cls=FlaskGroup, create_app=create_megaqc_app)
 def cli():
     """
-    Welcome to the MegaQC command line interface!\n MegaQC is built using the
-    Flask Python web framework.
+    Welcome to the MegaQC command line interface.
 
-    See below for the available commands - for example,
+    \nSee below for the available commands - for example,
```

Comment:
docformatter always adds a `.` to the top sentence no matter what.
This looked really ugly in the autogenerated docs then.

Hence, this change.